content/en/core/overview/pwa.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Installation as a Progressive Web App"
+linkTitle: "Progressive Web App"
+weight: 2
+description: >
+  What does it mean that the CHT Core web application is a Progressive Web App.
```

Comment:
```suggestion
  What does it mean that the CHT Core web application is a Progressive Web App?
```

Or `What it means that the CHT Core web application is a Progressive Web App.`  It just seems to word this as a question without the question mark.

Reply:
Changed to not be a question.